app/controllers/items_controller.rb

```diff
@@ -25,15 +25,23 @@ def new
   # POST /items
   # POST /items.json
   def create
-    @item = Item.new(item_params.merge(user_id: current_user.id))
+    @user = current_user
```

Comment:
`current_user` can be used in view templates if you prefer using that instead of `@user`